### DISCUSSION TITLE
Why checking latency with meme.kids?

### DISCUSSION THREAD

**Discussion by thomasmerz** [alexjustesen/speedtest-tracker#1433] [Q&A] [answered]
I found out by accident that every 20 minutes (according to my "Speedtest schedule") meme.kids has been in my Pi-hole log and I was wondering. So I did some investigation with `sudo ngrep meme.kids` and found this:

```
T 192.168.42.5:53876 -> 45.132.247.230:8080 [AP] #6751
  GET /speedtest/latency.txt?x=1715933402690 HTTP/1.1..Host: meme.kids:8080..Accept-Encoding: identity..User-Agent: Mozilla/5.0 (Linux-5.14.21-15
  0400.24.100-default-x86_64-with-glibc2.2.5; U; 64bit; en-us) Python/2.7.18 (KHTML, like Gecko) speedtest-cli/2.1.4b1....
```
Nothing is wrong, but I was just wondering WHY this domain and not something more "random"? 🤔

**Comment by alexjustesen** [comment]
That's an odd one and nothing I've seen in my logs. Are you using my image or LSIO?

**Reply by alexjustesen** [reply]
That site looks like someone's dashboard, double checking but it's not yours right?

**Reply by thomasmerz** [reply, marked as answer]
Must have been another "tool" that was confusing and misleading me. Sorry…
